src/map/lua/lua_baseentity.cpp

```diff
@@ -12395,6 +12395,28 @@ inline int32 CLuaBaseEntity::addFullGambit(lua_State* L)
     return 0;
 }
 
+inline int32 CLuaBaseEntity::trustPartyMessage(lua_State* L)
+{
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_TRUST);
+    TPZ_DEBUG_BREAK_IF(lua_isnil(L, 1) || !lua_isnumber(L, 1));
+
+    auto PTrust = static_cast<CTrustEntity*>(m_PBaseEntity);
+
+    auto p0 = lua_tointeger(L, 1);
+    auto p1 = 0;
+    auto p2 = 711;
+
+    auto PMaster = static_cast<CCharEntity*>(PTrust->PMaster);
+    PMaster->ForParty([&](CBattleEntity* PMember)
```

Comment:
Either the cast to PMaster, or PParty of PMaster is broken here while zoning.

Reply:
In the old lua-based system, the calls to :getParty() would fail for the same reason